sockeye/inference.py

```diff
@@ -340,6 +345,9 @@ def load_models(context: mx.context.Context,
                           "Source vocabulary ids do not match")
     utils.check_condition(all(set(vocab.items()) == set(target_vocabs[0].items()) for vocab in target_vocabs),
                           "Target vocabulary ids do not match")
+    utils.check_condition(all(m.batch_size == models[0].batch_size for m in models),
```

Comment:
why this check? You load all models a couple of lines above with the same batch_size so how can they differ?